### PR TITLE
Adding the ability for players to buy houses

### PR DESCRIPTION
**Pull request by gmriggs**

### CODE REVIEW

File: Source/ACE.Entity/Enum/Properties/PropertyFloat.cs [hunk 1]
```diff
@@ -191,7 +191,10 @@ public enum PropertyFloat : ushort
         [SendOnLogin]
         WeaponAuraElemental            = 170,
         [SendOnLogin]
-        WeaponAuraManaConv             = 171
+        WeaponAuraManaConv             = 171,
+        // custom
+        HouseBuyTimestamp              = 1000,
```
Comment (Mag-nus) [inline Source/ACE.Entity/Enum/Properties/PropertyFloat.cs]: For ACE Specific properties, we've been using the 9000+ range.

Comment (Mag-nus) [inline Source/ACE.Entity/Enum/Properties/PropertyFloat.cs]: For ACE specific properties, we've been using the 9000+ range.

File: Source/ACE.Server/Factories/WorldObjectFactory.cs [hunk 2]
```diff
@@ -199,7 +203,7 @@ public static List<WorldObject> CreateNewWorldObjects(List<LandblockInstance> so
             // TODO: Fix how objects are generated and linked
 
             // Code for new schema
-            foreach (var instance in sourceObjects.Where(x => x.IsLinkChild == false))
+            foreach (var instance in sourceObjects/*.Where(x => x.IsLinkChild == false)*/)
```
Comment (LtRipley36706) [inline Source/ACE.Server/Factories/WorldObjectFactory.cs]: This breaks links, should be restored before merge

File: Source/ACE.Entity/Enum/Properties/PropertyFloat.cs [hunk 3]
```diff
@@ -191,7 +191,10 @@ public enum PropertyFloat : ushort
         [SendOnLogin]
         WeaponAuraElemental            = 170,
         [SendOnLogin]
-        WeaponAuraManaConv             = 171
+        WeaponAuraManaConv             = 171,
+        // custom
+        HouseBuyTimestamp              = 9001,
```
Comment (LtRipley36706) [inline Source/ACE.Entity/Enum/Properties/PropertyFloat.cs]: PropertyInt.HousePurchaseTimestamp

**Comment by mcreedjr** [comment]
FWIW I play tested this and the functionality being merged in this PR (purchasing a dwelling) looks good to me. Nice and exciting work @gmriggs!

**Pull request merged**